resolwe_bio/tests/processes/test_alignment.py

```diff
@@ -228,29 +281,77 @@ def test_star(self):
             'star_sort': True,
         }
         aligned_reads = self.run_process('alignment-star', inputs)
-        self.assertFile(aligned_reads, 'bigwig', 'bigwig_star_ens_paired.bw')
+        self.assertFile(
+            aligned_reads,
+            'bigwig',
+            os.path.join('test_star', 'output', 'bigwig_star_ens_paired.bw')
+        )
         for data in Data.objects.all():
             self.assertStatus(data, Data.STATUS_DONE)
 
-        self.assertFile(aligned_reads, 'gene_counts', 'gene_counts_star_paired.tab.gz', compression='gzip')
+        self.assertFile(
+            aligned_reads,
+            'gene_counts',
+            os.path.join('test_star', 'output', 'gene_counts_star_paired.tab.gz'),
+            compression='gzip'
+        )
         exp = Data.objects.last()
-        self.assertFile(exp, 'exp', 'star_expression_paired.tab.gz', compression='gzip')
+        self.assertFile(
+            exp,
+            'exp',
+            os.path.join('test_star', 'output', 'star_expression_paired.tab.gz'),
+            compression='gzip'
+        )
         self.assertFields(exp, 'source', 'ENSEMBL')
-        self.assertFile(exp, 'exp_set', 'star_out_exp_set.txt.gz', compression='gzip')
-        self.assertJSON(exp, exp.output['exp_set_json'], '', 'star_exp_set.json.gz')
+        self.assertFile(
+            exp,
+            'exp_set',
+            os.path.join('test_star', 'output', 'star_out_exp_set.txt.gz'),
+            compression='gzip'
+        )
+        self.assertJSON(
+            exp,
+            exp.output['exp_set_json'],
+            '',
+            os.path.join('test_star', 'output', 'star_exp_set.json.gz')
```

Comment:
The file with the same name is tested in two places. For such cases, you cold simplify the lines by defining a variable first, and then using this where appropriate, e.g.
```
exp_set_json = os.path.join('test_star', 'output', 'star_exp_set.json.gz')

self.assertJSON(exp, exp.output['exp_set_json'], '', exp_set_json)
```